drizzlepac/sky.py

```diff
@@ -104,7 +104,6 @@ def sky(input=None,outExt=None,configObj=None, group=None, editpars=False, **inp
         run(configObj,outExt=outExt)
 
 
-#this is the function that will be called from TEAL
```

Comment:
Similar comments here as made in other files.  Either leave the comment and add further documentation or remove comment and functionality.